packages/web/src/apollo/index.tsx

```diff
@@ -26,10 +31,60 @@ import { GraphQLHooksProvider } from '../components/GraphQLHooksProvider'
 
 export type ApolloClientCacheConfig = apolloClient.InMemoryCacheConfig
 
+export type GraphQLClientConfigProp = Omit<
+  ApolloClientOptions<unknown>,
+  'cache' | 'link'
+> & {
+  cache?: ApolloCache<unknown>
+  /**
+   * Configuration for Apollo Client's `InMemoryCache`.
+   * See https://www.apollographql.com/docs/react/caching/cache-configuration/.
+   */
+  cacheConfig?: ApolloClientCacheConfig
+  /**
+   * Configuration for the terminating `HttpLink`.
+   * See https://www.apollographql.com/docs/react/api/link/apollo-link-http/#httplink-constructor-options.
+   */
+  httpLinkConfig?: apolloClient.HttpOptions
```

Comment:
Perhaps add the example that this PR is intended to help with -- that is, the example of using the config to set the credentials policy so that cookies can be sent to other domains?

```
<RedwoodApolloProvider graphQLClientConfig={{ 
  httpLinkConfig: { credentials: 'include' } 
}}>
```